src/commonMain/kotlin/exchange.dydx.abacus/state/v2/manager/StateManagerAdaptorV2.kt

```diff
@@ -429,6 +445,28 @@ internal class StateManagerAdaptorV2(
         return null
     }
 
+    private fun fetchGeo() {
+        val url = "https://api.dydx.exchange/v4/geo"
```

Comment:
Sounds good, I added to environment.
It would have the same cloudflare restriction as FE I think, so FE shouldn't be able to hit it. which will only be limited to OFAC countries after this compliance project is released.